Update security guideline documentation (#6029)
wso2/docs-is: en/identity-server/7.1.0/docs/deploy/security/configure-transport-level-security.md

2. Restart the server.

## Configure cipher suites

> By default, all SSL ciphers supported by JSSE are enabled. To restrict the server to a specific set of secure cipher suites, explicitly configure the `ciphers` property. If left blank, weak ciphers (including EXPORT ciphers) will also be available, which can make the server vulnerable to attacks such as the Logjam attack.

Reviewer: ```suggestion
By default, {{product_name}} allows all SSL ciphers supported by JSSE. To restrict the server to a specific set of secure cipher suites, you can configure the `ciphers` property. If left blank, weak ciphers (including EXPORT ciphers) will also be available, which can make the server vulnerable to attacks such as the Logjam attack.
```

Reviewer: Please add reference links for JSSE and Logkam attack.

Author: These are outdated now. Will remove
